### PR TITLE
Use only localhost for IPC + add camera service.

### PR DESCRIPTION
**Pull request by ethanrublee**

### CODE REVIEW

File: python/farm_ng/ipc.py [hunk 1]
```diff
@@ -212,7 +215,7 @@ def _make_mc_send_socket(self):
 
         # Set the time-to-live for messages to 1 so they do not
```
Comment (ethanrublee) [inline python/farm_ng/ipc.py:216]: Update comment.

File: jetson/install.sh [hunk 2]
```diff
@@ -9,6 +9,10 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 export SERVICE_DIR=$( cd "$( dirname "${SOURCE}" )" >/dev/null 2>&1 && pwd )
 
+# clean up old systemctl services
+rm -f /etc/systemd/system/multi-user.target.wants/tractor*
```
Comment (ethanrublee) [inline jetson/install.sh:13]: @isherman need to add this or similar for removing the enabled services.

**Comment by isherman** [comment]
Plan for this PR:

**Comment by isherman** [comment]
Still testing locally

**Comment by isherman** [comment]
@ethanrublee this is ready for your review. Tried to unify the go/py/cpp ipc logic around how they handle announcements. Also includes some minor improvements to the structure of the Go code, and your changes to the systemd `.path` files. I've been testing successfully locally with a `make` and several reboots, but could use your help testing too.

**Comment by ethanrublee** [comment]
Sorry, reviewed it as farm-ng-bot...

**Comment by ethanrublee** [comment]
Tested on my tractor, going to merge.

**Pull request merged**